$ diff -Nub B/mothur-1.48.0/source/uchime_src/makefile A/mothur-1.48.0/source/uchime_src/makefile 
--- B/mothur-1.48.0/source/uchime_src/makefile	2022-05-18 17:27:36.000000000 +0100
+++ A/mothur-1.48.0/source/uchime_src/makefile	2023-10-04 11:39:44.418459181 +0100
@@ -1,4 +1,4 @@
-CXXFLAGS = -O3 -D_FILE_OFFSET_BITS=64 -DNDEBUG=1 -DUCHIMES=1
+CXXFLAGS = -std=c++11 -O3 -D_FILE_OFFSET_BITS=64 -DNDEBUG=1 -DUCHIMES=1
 LDFLAGS = -g
 
 #
